packages/block-editor/src/components/responsive-block-control/index.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * External dependencies
+ */
+import { isFunction } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { __, _x, sprintf } from '@wordpress/i18n';
+
+import { Fragment, useState, useEffect, useCallback } from '@wordpress/element';
+
+import {
+	ToggleControl,
+} from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import ResponsiveBlockControlLabel from './label';
+
+function ResponsiveBlockControl( props ) {
+	const {
+		legend,
+		property,
+		toggleLabel,
+		onIsResponsiveModeChange,
+		renderDefaultControl,
+		renderResponsiveControls,
+		responsiveControlsActive = false,
+		defaultLabel = {
+			id: 'all',
+			label: _x( 'All', 'Label. Used to signify a layout property (eg: margin, padding) should apply uniformly to all screensizes.' ),
+		},
+		devices = [
+			{
+				id: 'small',
+				label: __( 'Small devices' ),
+			},
+			{
+				id: 'medium',
+				label: __( 'Medium devices' ),
+			},
+			{
+				id: 'large',
+				label: __( 'Large devices' ),
+			},
+		],
+	} = props;
+
+	const [ isResponsiveMode, setIsResponsiveMode ] = useState( responsiveControlsActive );
+
+	useEffect( () => {
+		if ( isFunction( onIsResponsiveModeChange ) ) {
+			onIsResponsiveModeChange( isResponsiveMode );
+		}
+	}, [ isResponsiveMode ] );
+
+	const handleToggle = useCallback( ( isChecked ) => {
+		setIsResponsiveMode( ! isChecked );
+	} );
+
+	if ( ! legend || ! property || ! renderDefaultControl ) {
+		return null;
+	}
+
+	const toggleControlLabel = toggleLabel || sprintf( _x( 'Use the same %s on all screensizes.', 'Toggle control label. Should the property be the same across all screen sizes or unique per screen size.' ), property );
```

Comment:
We need a translator comment.
An example of a comment is `/* translators: accessibility text. %1: current block position (number). %2: next block position (number) */`

Reply:
Got it. I totally misunderstood this before.

I've resolved this, but I have noticed that I cannot see this type of comment mentioned anywhere in the `docs`. Did I miss something or do we need to add this?